docs/reference/index.rst

```diff
@@ -36,3 +36,15 @@ Diagram Visualization
     persim.plot_diagrams
     persim.bottleneck_matching
     persim.wasserstein_matching
+
+
+
+    
+Persistence barcode measure
+======================
```

Comment:
Have you tried rendering the docs? I think you'll need to add more "=" until there are enough to completely cover the text.